src/test/kotlin/com/exactpro/epfast/template/dsl/SequenceBuilder.kt

```diff
@@ -33,10 +34,65 @@ class SequenceBuilder internal constructor(
         field.length = LengthFieldBuilder().apply(block).field
     }
 
-    fun instructions(block: InstructionsBuilder.() -> Unit) {
-        InstructionsBuilder(field.instructions).apply(block)
+    fun length(name: String, block: LengthFieldBuilder.() -> Unit) {
+        field.length = LengthFieldBuilder().also { it.name = name }.apply(block).field
     }
+
+    fun instructions(block: InstructionsBuilder.() -> Unit) =
+        InstructionsBuilder(field.instructions).apply(block)
+
+    fun asciiString(
+        name: String,
+        namespace: String = DEFAULT_NAMESPACE,
+        block: AsciiStringFieldBuilder.() -> Unit = {}
```

Comment:
Default value for block should be added for everything but instructions, sequence, template and similar entities which don't make sense without body